.github/workflows/main.yml

```diff
@@ -223,7 +223,7 @@ jobs:
       - uses: ./.github/actions/rustup
       - uses: ./.github/actions/sccache
         with:
-          key: ${{ matrix.os }}-${{ matrix.feature }}
+          key: ${{ matrix.os }}-${{ matrix.feature }}-1.0
```

Comment:
@mothran @morgangibbs87 It looks like we have some sccache invalidation issues. We need this to apparently force the cache between different versions to be distinct. Otherwise we get weird linking issues which makes it seem like the CUDA caching isn't working right.

Reply:
Interesting, I will double check out ssccache based builds in our docker agent builds since I think that builds the kernels.